lib/robot.js

```diff
@@ -142,7 +148,7 @@ class Robot {
         return this.app.createToken(id)
       }, {ttl: 60 * 59}) // Cache for 1 minute less than GitHub expiry
 
-      github = new GitHubApi({debug: process.env.LOG_LEVEL === 'trace'})
+      github = new GitHubApi(githubOptions)
       github.authenticate({type: 'token', token: res.data.token})
     } else {
       github = await this.app.asApp()
```

Comment:
We're going to need to pass in the options to this call as well. This is defined in https://github.com/probot/github-app/blob/431d8d24a019d93be6a3e6ed28d95ae96eb0f063/index.js#L5-L10

I don't really like this separate github-app module, so I'd be interested in just copying the code into Probot and modifying from there. 😞

Reply:
Oh I missed this. I do agree with you on just moving github-app into probot. I'd be inclined to do that as a separate PR though, to be merged before this one.

Reply:
I've opened up #333 to do this; I'll merge it into this PR and make the necessary adjustments, but ideally #333 should be merged to `master` first, followed by the necessary conflict resolution with this PR.